src/MicrosoftTeams.ts

```diff
@@ -84,7 +84,8 @@ export const enum HostClientType {
   desktop = "desktop",
   web = "web",
   android = "android",
-  ios = "ios"
+  ios = "ios",
```

Comment:
Please sync latest changes from master and move your changes to ./src/public/constants.ts file and ./src/public/authentication.ts

Reply:
Just updated the PR, also updated the comment in interface.ts, just fyi.

Reply:
Sorry probably should have merged instead of trying to rebase.